sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -199,12 +167,13 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     institutionModel: InstitutionFormModel,
     userInfo: UserInfo,
   ): Promise<Institution> {
+    // New user to be associated with the institution. It will also be considered the audit user.
```

Comment:
i didn't get this point `It will also be considered the audit user`

Reply:
This is the process that happens when a new BCeID business user is also creating the institution.
Basically at this point, the `userToken` does not have the `userId` because the user is not on the DB yet.
So, the user being created here will be the user also to be used as the audit user. Does it sound better now?

Reply:
oh,, okay..got it